### ISSUE TITLE
Zoom toolbar for feature diagrams not readable

### ISSUE THREAD

**Issue by tthuem** [FeatureIDE/FeatureIDE#363]
![zwischenablage-2](https://cloud.githubusercontent.com/assets/5330357/16579823/dac39fe6-42a1-11e6-94ea-f112ff5fe43f.png)

**Comment by Kogoro** [comment]
The bug also occurs in eclipse Neon [Release (4.6.0), Build id: 20160613-1800], so it should be some issue on our side.

**Comment by Kogoro** [comment]
The zoom element looks okay on Ubuntu 16.04 with Neon (GTK 2) and [Solus](https://solus-project.com/) on Neon.1. Looks like to solve this issue under Windows, we must change the toolbar height in an own theme and ship the theme with FeatureIDE.
![zoombar_ubuntu](https://cloud.githubusercontent.com/assets/8019776/19065885/82359d0e-8a16-11e6-80ec-8b61d37cc57d.png)

**Comment by tthuem** [comment]
Why is there a need to ship a theme? Rather sounds like a hack. Is there no general solution in the Eclipse world for that?

**Comment by Kogoro** [comment]
I looked further. Seems that it is SWT instead of GEF and is an issue that eclipse can't calculate the height of the toolbar ([Bug report](https://bugs.eclipse.org/bugs/show_bug.cgi?id=183003)).

The report also contains a workaround, but it will add a not visible image to the toolbar to set the height for all elements. I pushed the working version for a review to the corresponding branch.

The result will looks like this:
<img width="960" alt="2016-10-09_01-54-15" src="https://cloud.githubusercontent.com/assets/8019776/19216614/912c6dfa-8dc3-11e6-985d-46883b0cfe90.png">
